syntax = "proto3";
package com.dxc.ptinsight.analytics;

import "ptinsight/common/proto/base.proto";
import "google/protobuf/timestamp.proto";


option java_package = "com.dxc.ptinsight.proto.analytics";


message VehicleCount {
  int64 geocell = 1;
  int32 count = 2;
}

message FinalStopCount {
  int64 geocell = 1;
  int32 count = 2;
}
